### PR TITLE
Add com.ibm.ws.jaxws.X.wsat_fat to Open-Liberty

### PR DESCRIPTION
**Pull request by neuwerk**
This is a migration of the `com.ibm.ws.jaxws.wsat_fat` to Open-Liberty, which should have been done along side the other buckets. It's been renamed however to indicate that this bucket will now run against multiple versions of the JAX-WS/XML-WS features. 

Enabling of reruns against other features will be added to this bucket in a separate pull request.

### CODE REVIEW

**Comment by neuwerk** [comment]
#build

File: dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsClient1/src/com/ibm/ws/policyattachments/client1/ClientServlet1.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
+package com.ibm.ws.policyattachments.client1;
```
Comment (BerksanAtes) [inline dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsClient1/src/com/ibm/ws/policyattachments/client1/ClientServlet1.java:11]: Do we need to add copyright?

File: dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsClient2/.settings/org.eclipse.jdt.core.prefs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
```
Comment (BerksanAtes) [inline dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsClient2/.settings/org.eclipse.jdt.core.prefs]: Java  1.8?

File: dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsService1/.settings/org.eclipse.jdt.apt.core.prefs [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+org.eclipse.jdt.apt.processorOptions/-d=C\:\\Jordan\\workspace\\WDT\\policyAttachmentsService1\\WebContent\\WEB-INF\\classes
+org.eclipse.jdt.apt.processorOptions/-s=C\:\\Jordan\\workspace\\WDT\\policyAttachmentsService1\\.apt_generated
+org.eclipse.jdt.apt.processorOptions/-source=1.7
```
Comment (BerksanAtes) [inline dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsService1/.settings/org.eclipse.jdt.apt.core.prefs]: Java 1.8?

File: dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsService1/.settings/org.eclipse.jdt.core.prefs [hunk 4]
```diff
@@ -0,0 +1,8 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
```
Comment (BerksanAtes) [inline dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsService1/.settings/org.eclipse.jdt.core.prefs]: java 1.8?

File: dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsService2/.settings/org.eclipse.jdt.core.prefs [hunk 5]
```diff
@@ -0,0 +1,8 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
```
Comment (BerksanAtes) [inline dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsService2/.settings/org.eclipse.jdt.core.prefs]: Java 1.8?

File: dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsService3/.classpath [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+<classpath>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path=".apt_generated">
+		<attributes>
+			<attribute name="optional" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.jst.j2ee.internal.web.container"/>
+	<classpathentry kind="con" path="org.eclipse.jst.j2ee.internal.module.container"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7">
```
Comment (BerksanAtes) [inline dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsService3/.classpath]: Java 1.8?

File: dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsService3/.settings/org.eclipse.jdt.core.prefs [hunk 7]
```diff
@@ -0,0 +1,8 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
```
Comment (BerksanAtes) [inline dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/policyAttachmentsService3/.settings/org.eclipse.jdt.core.prefs]: Java 1.8?

File: dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/simpleTestService/src/com/ibm/ws/jaxws/wsat/testservice/client/SimpleServiceServlet.java [hunk 8]
```diff
@@ -0,0 +1,149 @@
+package com.ibm.ws.jaxws.wsat.testservice.client;
```
Comment (BerksanAtes) [inline dev/com.ibm.ws.jaxws.X.wsat_fat/test-applications/simpleTestService/src/com/ibm/ws/jaxws/wsat/testservice/client/SimpleServiceServlet.java:11]: Copyright?

**Review by BerksanAtes — APPROVED** [review]
Looks great!

**Comment by neuwerk** [comment]
#run-libby-bot

**Pull request merged**